accepted/2021/nano-framework-tfm/nan-framework-tfm.md

```diff
@@ -0,0 +1,61 @@
+# Nano Framework TFM
+
+**Owner** [Immo Landwerth](https://github.com/terrajobst)
+**Owner** [Laurent Ellerbach](https://github.com/Ellerbach)
+**Owner** [José Simões](https://github.com/josesimoes)
+
+While .NET has been optimized to be a general purpose application development
+framework, we also created a flavor of .NET that can be used in the context of
+very resource-constrained hardware, called [.NET Micro Framework][netmf]. To
+give you an idea what resource-constrained looks like: 256 KB of flash and 64 KB
```

Comment:
For sake of accuracy: these are the constrains of the .NET nanoFramework.
On the old .NETMF these were around 512kB of flash and 128kB of RAM.